third_party/txt/src/txt/paragraph.cc

```diff
@@ -1458,12 +1461,17 @@ std::vector<Paragraph::TextBox> Paragraph::GetRectsForRange(
                            box.direction);
       }
     } else if (rect_height_style == RectHeightStyle::kStrut) {
-      for (const Paragraph::TextBox& box : kv.second.boxes) {
-        boxes.emplace_back(
-            SkRect::MakeLTRB(
-                box.rect.fLeft, line_baselines_[kv.first] - strut_.ascent,
-                box.rect.fRight, line_baselines_[kv.first] + strut_.descent),
-            box.direction);
+      if (IsStrutValid()) {
```

Comment:
Can you document this fallback behavior here as well as in the enum declaration?